common/djangoapps/util/course.py

```diff
@@ -0,0 +1,49 @@
+"""
+Utility methods related to course
+"""
+import logging
+from django.conf import settings
+
+from opaque_keys.edx.keys import CourseKey
+import re
+
+log = logging.getLogger(__name__)
+
+
+def get_lms_link_for_about_page(request, course_key):
+    """
+    Returns the url to the course about page from the location tuple.
+    """
+
+    assert isinstance(course_key, CourseKey)
+
+    if settings.FEATURES.get('ENABLE_MKTG_SITE', False):
+        if not hasattr(settings, 'MKTG_URLS'):
+            log.exception("ENABLE_MKTG_SITE is True, but MKTG_URLS is not defined.")
+            return None
+
+        marketing_urls = settings.MKTG_URLS
+
+        # Root will be "https://www.edx.org". The complete URL will still not be exactly correct,
+        # but redirects exist from www.edx.org to get to the Drupal course about page URL.
+        about_base = marketing_urls.get('ROOT', None)
+
+        if about_base is None:
+            log.exception('There is no ROOT defined in MKTG_URLS')
+            return None
+
+        # Strip off https:// (or http://) to be consistent with the formatting of LMS_BASE.
+        about_base = re.sub(r"^https?://", "", about_base)
+
+    elif hasattr(settings, "LMS_BASE"):
```

Comment:
@wajeeha-khalid so in case of lms app, settings object has no attribute **LMS_BASE**, that's why you are using _hasattr_.
instead of making this check specific to LMS or CMS, is it possible to use the value of a common variable that exists in both LMS and CMS ?
Not sure about **ENV_TOKENS** in envs/aws.py settings but seems like this will have the same value for both apps, can we use that value here?
cc: @jcdyer